Dfe.Academies.External.Web/AcademiesAPIResponseModels/Trusts/TrustSearchDto.cs

```diff
@@ -2,6 +2,19 @@
 
 public record TrustSearchDto
 {
+	public TrustSearchDto(string ukprn,
```

Comment:
We've been using the constructor-only syntax for records, which looks like:

```
public record Person(string FirstName, string LastName);
```

Just a thought
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record

Reply:
I need the DisplayName prop though for the autocomplete